diff --git a/src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirApiWorkFlowServiceImpl.java b/src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirApiWorkFlowServiceImpl.java
index 955c28a..c26b807 100644
--- a/src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirApiWorkFlowServiceImpl.java
+++ b/src/main/java/org/openelisglobal/dataexchange/fhir/service/FhirApiWorkFlowServiceImpl.java
@@ -53,7 +53,6 @@ import org.openelisglobal.referral.service.ReferralService;
 import org.openelisglobal.spring.util.SpringContext;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.scheduling.annotation.Async;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
@@ -86,13 +85,12 @@ public class FhirApiWorkFlowServiceImpl implements FhirApiWorkflowService {
     private Optional<Boolean> remoteStoreUpdateStatus;
 
     @Override
-    @Scheduled(initialDelay = 10 * 1000, fixedRateString = "${org.openelisglobal.remote.poll.frequency:120000}")
+    @Scheduled(initialDelay = 10 * 1000, fixedDelayString = "${org.openelisglobal.remote.poll.frequency:120000}")
     public void pollForRemoteTasks() {
         processWorkflow(ResourceType.Task);
     }
 
     @Override
-    @Async
     public void processWorkflow(ResourceType resourceType) {
         for (String remoteStorePath : fhirConfig.getRemoteStorePaths()) {
             switch (resourceType) {
